src/components/NetworkHealthIndicator.tsx

```diff
@@ -66,143 +64,125 @@ const overallConnectionStateColor: Record<NetworkHealthState, IconColorConfig> =
     },
   }
 
-const overallConnectionStateIcon: Record<
-  NetworkHealthState,
-  ActionIconProps['icon']
-> = {
+const overallConnectionStateIcon = {
   [NetworkHealthState.Ok]: 'network',
   [NetworkHealthState.Weak]: 'network',
   [NetworkHealthState.Issue]: 'networkCrossedOut',
   [NetworkHealthState.Disconnected]: 'networkCrossedOut',
+} as const
+
+export const useNetworkHealthStatus = (): StatusBarItemType => {
+  const { overallState } = useNetworkContext()
+
+  return {
+    id: 'network-health',
+    'data-testid': `network-toggle-${
+      overallState === NetworkHealthState.Ok ? 'ok' : 'other'
+    }`,
+    label: `Network health (${NETWORK_HEALTH_TEXT[overallState]})`,
+    hideLabel: true,
+    element: 'popover',
+    className: overallConnectionStateColor[overallState].icon,
```

Comment:
I like the change here from bg color to icon color!

Reply:
I think this is a great compromise @lee-at-zoo-corp, since we keep the ability to know on screenshots which state it was, but it will be less distracting than a bg color change?